src/sys/resource.rs

```diff
@@ -0,0 +1,122 @@
+//! Configure the process resource limits.
+use std::mem;
+
+use libc::{self, c_int, rlimit, RLIM_INFINITY};
+pub use libc::rlim_t;
+
+use {Errno, Result};
+
+libc_enum!{
+    #[repr(i32)]
+    pub enum Resource {
+        // POSIX
```

Comment:
Please provide doc comments for every entry here to give users an understanding of what they mean.

Reply:
Done